api/src/pcapi/connectors/serialization/ems_serializers.py

```diff
@@ -33,3 +33,37 @@ class Site(BaseModel):
 class CinemasProgramsResponse(BaseModel):
     sites: list[Site]
     version: int
+
+
+class Ticket(BaseModel):
+    num_caisse: str
+    code_barre: str
+    num_trans: int
+    num_ope: int
+    code_tarif: str
+    num_serie: int
+    montant: float
+    num_place: str
+
+
+class ReservationPassCultureRequestSerializer(BaseModel):
+    num_cine: str  # VenueProvider.venueIdAtOfferProvider
+    id_seance: str  # show_id
+    qte_place: int
+    pass_culture_price: float
+    total_price: float
+    email: str
+    num_pass_culture: str  # User.id
+
+
+class ReservationPassCultureResponseSerializer(ReservationPassCultureRequestSerializer):
```

Comment:
Pareil que le commentaire précédent 
```suggestion
class ReservationPassCultureResponse(ReservationPassCultureRequestSerializer):
```